docker-compose.prod.yml

```diff
@@ -0,0 +1,103 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:${DATE_POSTGRESQL_VERSION}-alpine
+    restart: always
+    volumes:
+      - date_postgres_data:/var/lib/postgresql/data/
+    ports:
+      - ${DATE_DB_PORT}:5432
+    environment:
+      - POSTGRES_PASSWORD=${DATE_DB_PASSWORD}
+  web:
+    image: ghcr.io/datateknologerna-vid-abo-akademi/date-website:${DATE_IMG_TAG}
+    restart: always
+    command: /bin/bash -c "./wait-for-postgres.sh db:5432 && python /code/manage.py migrate --noinput && python manage.py collectstatic --noinput && gunicorn -w 3 --bind=0.0.0.0 core.wsgi"
+    environment:
+      - SECRET_KEY=${DATE_SECRET_KEY}
+      - DEBUG=${DATE_DEBUG}
+      - DEVELOP=${DATE_DEVELOP}
+      - DB_PASSWORD=${DATE_DB_PASSWORD}
+      - USE_S3=${USE_S3}
+      - AWS_S3_ENDPOINT_URL=${S3_ENDPOINT_URL}
+      - AWS_ACCESS_KEY_ID=${S3_ACCESS_KEY}
+      - AWS_SECRET_ACCESS_KEY=${S3_SECRET_KEY}
+      - AWS_STORAGE_BUCKET_NAME=${S3_BUCKET_NAME}
+      - PUBLIC_MEDIA_LOCATION=${PUBLIC_MEDIA_LOCATION}
+      - PRIVATE_MEDIA_LOCATION=${PRIVATE_MEDIA_LOCATION}
+      - ALLOWED_HOSTS=${ALLOWED_HOSTS}
+      - ALLOWED_ORIGINS=${ALLOWED_ORIGINS}
+      - EXTRA_STAFF_GROUPS=${EXTRA_STAFF_GROUPS}
+      - EMAIL_HOST_RECEIVER=${EMAIL_HOST_RECEIVER}
+      - DEFAULT_FROM_EMAIL=${DEFAULT_FROM_EMAIL}
+      - EMAIL_HOST=${EMAIL_HOST}
+      - EMAIL_HOST_USER=${EMAIL_HOST_USER}
+      - EMAIL_HOST_PASSWORD=${EMAIL_HOST_PASSWORD}
+      - CF_TURNSTILE_SITE_KEY=${CF_TURNSTILE_SITE_KEY}
+      - CF_TURNSTILE_SECRET_KEY=${CF_TURNSTILE_SECRET_KEY}
```

Comment:
Consider using env_file directive with separate env files for each container if needed. Could make the compose files cleaner. "web" and "asgi" seems to have the same env vars defined so they could maybe have a common env file that's defined with the env_file directive.

Reply:
Topic for a later issue, problem is some variable names are different for host and container and all env variables are currently sourced into host env first. 

Probably should be tackled in #348

Reply:
We could also start using something like doppler for secrets.
https://www.doppler.com/

Reply:
Free subscription for students :)
All the env variables should also have default values in the app, e.g. `.get(EMAIL_HOST_USER, 'admin@example.com')`